Script/dateController.js

```diff
@@ -3,14 +3,36 @@ $(document).ready(function()
   loadDates();
 });
 
+function addZero(i)
+{
+  if (i < 10) { i = "0" + i; } return i; 
+}
+
+function tidyUpTimestamp(timestamp) 
+{
+    var dateCreated = new Date(timestamp);
+    dateCreated.toString();
+    var date = dateCreated.getDate();
+    var month = dateCreated.getMonth();
+    var year = dateCreated.getFullYear();
+    var hours = dateCreated.getHours();
+    var minutes = addZero(dateCreated.getMinutes());
```

Comment:
No need to make a one line function, apply the function in the minute var using ternary operator
var minute = (dateCreated.getMinutues() <  10) ? i = "0" + i : i;